lib/core/client.js

```diff
@@ -739,6 +762,63 @@ function connect (client) {
     .on('close', onSocketClose)
 }
 
+function createName (url, tls) {
+  let name = `${url.hostname}:${url.port}`
+  if (tls) {
+    name += ':'
+    if (tls.ca) { name += tls.ca }
+
+    name += ':'
+    if (tls.cert) { name += tls.cert }
+
+    name += ':'
+    if (tls.clientCertEngine) { name += tls.clientCertEngine }
```

Comment:
I guess this comes from Node core? Can we keep with the undici style, i.e. not single line `{ statement }`?